<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>471</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Matricola</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dip_matricola"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nome      </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dip_nome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Cognome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dip_cognome"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Data Nascita</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dip_data_nascita"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Genere</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="dip_genere"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Citta Nascita</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_citta_nascita"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Codice Fiscale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_codice_fiscale"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Indirizzo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_indirizzo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cap</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_cap"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_21">
      <item>
       <widget class="QLabel" name="label_21">
        <property name="text">
         <string>Comune</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_comune"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_22">
      <item>
       <widget class="QLabel" name="label_22">
        <property name="text">
         <string>Provincia</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_provincia"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_24">
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>Titolo di Studio</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_titolo_studio"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_27">
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Funzione</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_funzione"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_26">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Ruolo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combobox_ruolo">
          <property name="maximumSize">
           <size>
            <width>229</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_28">
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Email Aziendale</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_email_aziendale"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_31">
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Telefono</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="dip_telefono"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_32">
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Stato</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combobox_stato">
        <property name="maximumSize">
         <size>
          <width>229</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>660</y>
     <width>444</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_33">
    <item>
     <widget class="QPushButton" name="button_cancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_delete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_curricula">
      <property name="text">
       <string>Curriculum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_save">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
